### ISSUE TITLE
Internal Page Headers Not Accessible

### ISSUE THREAD

**Issue by rachelnicole** [tstanley93/waf-portal#1]
For future development after the proof of concept is completed, the way the menus are getting loaded need to be fixed for pages like applications, logs, signup, violations, and visualize.

It's currently a menu image that utilizes javascript to load a div block for the menu items.

I'd recommend switching to a svg icon that's well annotated with text tags, and to turn the div block into a list.
